image-layout.md

```diff
@@ -1,13 +1,13 @@
 ## Open Container Initiative Image Layout Specification
 
-The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) references (refs).
+The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) reference tags (tags).
```

Comment:
On Mon, Aug 29, 2016 at 03:24:32PM -0700, Brandon Philips wrote:
> Eh, I think that it is going to be more common for someone to download "an
> entire image named foo" then wanting to import the whole thing and expect
> the tags to show back up rather than forcing the user to specify every ref.
> For example you would actually need to do:
> 
> my-tool import /path/to/layout my-img-foo-bar-v1.0.0 my.img/foo/bar v1.0.0
> my-tool run my.img/foo/bar:v1.0.0
> 
> vs
> 
> my-tool import /path/to/layout  my.img/foo/bar
> my-tool run my.img/foo/bar:v1.0.0
> 
> I feel there is a usability cost to supporting lots of imgs in a single
> layout on this common import UX.

So add an:

  $ my-tool import-all LAYOUT-PATH

that imports all the refs (still getting names from the CAS blobs
[1]).  Or, without a HEAD spec, make:

  $ my-tool import LAYOUT-PATH

(without an explicit REF) import all the refs.

Specifying an unverified name on the command line or extracting
information from *the ref name* (vs. the referenced CAS content)
should not be part of your main UX.

[1]: https://github.com/opencontainers/image-spec/pull/219#discussion_r76698336